### ISSUE TITLE
Referring to wikidata

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#2065]
Dear @thea-m @DenisNosnitsin1970 
I was having second thoughts on using just the wd references instead of internal records - possibly unnecessary but still...
If we have an encoding like, (BSLet169)
```
<provenance>
                        The ms. was written in <placeName ref="LOC1310Aksum">Aksum</placeName> 
                        and presented by <persName ref="PRS10803Yeshaq"/> 
                        to the Ethiopian community in <placeName ref="wd:Q1218">Jerusalem</placeName>. 
                        From there it passed to <placeName ref="wd:Q220">Rome</placeName> 
                        together with other mss. and, having been transferred from the hostel of  <placeName ref="INS0302SanStefano">San Stefano dei Mori</placeName> 
                        to the <placeName ref="INS0307MB">Museo Borgiano</placeName>, 
                        it was stolen and brought to  <placeName ref="wd:Q84">London</placeName>.
                        <p>In <date>1817</date> the bookseller <persName ref="PRS10804Smith">J. Smith</persName> 
                           in Drury Lane sold it to Church Missionary Society, which transferred it to the British and Foreign Bible Society, for use in a printed Ethiopic edition. </p>
                           </provenance>
```
etc etc for the moment there is no way on seeing e.g. all mentions of London or Rome (which I would get in the list of attestations in the respective records if the `bm ``ref `were used).
Is it something we can live without? Nothing we are losing?

And another thing, when we encode, as e g in INS0972MahdaraSebhat,  `<settlement ref="wd:Q3624"/>`, then the reference for the manuscripts starts with Addis Ababa (`en `of wd) instead of ʾAddis ʾAbabā. Maybe not important but well I find it confusing.

So I do wonder if it is better to have local IDs for `ref`, and only in the record itself do the `sameAs`=wd connection...

@thea-m attended several meetings on authority files, but I think aligning through sameAs is probably the main part of this.

(potentially one could also work on the code to show attestations not only for the entry itself but also for the ID linked through sameAs attribute, I guess this could be done, but not sure by me - and then the spelling issue remains)

**Comment by DenisNosnitsin1970** [comment]
Sorry, I do not understand - if it is recored like this, we will not see all mentions of London and Rome - where? In search results? The second issue, of orthography, may affect cataloguers and users. Personally I would prefer to use local IDs, using wd instead seems to me not very practical (esp. if the information is already in the record).

**Comment by eu-genia** [comment]
I mean if you open Rome or Jerusalem on Bm you won't be seeing under "show attestations" the manuscript from the above example as the links point to wd.

**Comment by DenisNosnitsin1970** [comment]
I see. Of course, I would need rather manuscripts than the wd link.

**Comment by thea-m** [comment]
Let us think whether there would be any disadvantages for using Bm IDs in these cases. 
This would mean that we would potentially have to create more IDs for places outside the scope of the manuscript culture of Ethiopia and Eritrea. Is this a conceptual problem?
It could also potentially take more time, but I actually don't think that the difference would be very big, there are probably not that many places involved.
It would also mean a change from the project's past decisions, which have been explained and published, i.e. https://www.fdr.uni-hamburg.de/record/269#.YlPklMhBw2w. I don't see that as a hindrance (it might even be worthwhile to publish somewhere a small paper dealing with changes of encoding in a long-term project), but it would mean that the decision and possible change of encoding should be well documented. 
Do any other potential problems or disadvantages come to your minds?

(Unrelated, but important nota bene: What I undestood from my involvement in the authority files meetings is that the crucial point is that our IDs are added to wikidata. This has to be done manually. As I did it a few weeks ago for all places, institutions and persons with wikidata in their records, it's very important that we all remember to do this whenever we 1) add wikidata to a record where there was none or 2) create a new record with wikidata.)

**Comment by eu-genia** [comment]
The article quoted does declare alignment but it does use local IDs for Dayr as Suryan and Gondar, so I see no contradiction. Alignment is achieved by the sameAs linking. It actually goes against the article to use wd ID for Gondar or Addis, what has happened frequently, even for places within Ethiopian and Eritrean borders. 
I also believe that most places do have internal IDs, probably not many would have to be added, we simply all have to work along the same rules, currently there are many inconsistencies.
Yes of course many wd entries have to be created in any case...

**Comment by DenisNosnitsin1970** [comment]
Not a true conceptual or technical problem, but rather a problem in terms of time and work investment, and consistency.

**Comment by DenisNosnitsin1970** [comment]
Does it mean we will have to work much with wiki?

**Comment by eu-genia** [comment]
Well yes this was already decided time ago, we should, yes, to be able to work internationally. All our places and persons should ideally get a wikidata ID.

**Comment by DenisNosnitsin1970** [comment]
ok

**Comment by thea-m** [comment]
I am going through the guidelines to update them. I suppose the same logic applies to references using pleiades instead of wikidata. They are not many, I found only one in places and in manuscripts only in the inscription RIE271and190. 
I think @sologebre 's opinion would be valuable. Could you be so kind and read through this issue? Would there be something crucial lost by substituting the pleiades references by internal Bm IDs?

**Comment by eu-genia** [comment]
Yes this is the same issue from our point of view; the link in RIE to pleiades:39303 is actually to Aksum, but of course this attestation is not visible from our Aksum record.
Can one have more than one sameAs alignments? Where are pleiades IDs otherwise encoded? So that not to lose information?

**Comment by Dirbwork** [comment]
Yes, I think so. There are cases where we used pleiades id without using Bm IDs, especially for biblical places, places outside the Ethiopia and Eritrea domain, associated with different projects working on ancient places. Note that we always use Bm ID for places in the Ethiopia and Eritrea domain. However, using Bm ID for places that are not in our domain can be redundant. In this case, we can use either the wikdata id or the pleiades id depending on the information we have, or both . So we should consider these things when we substitute pleiades id with BM; yes, it affects the information in the texts we have encoded so far.

**Comment by sologebre** [comment]
Yes, I think so. There are cases where we used pleiades id without using Bm IDs, especially for biblical places, places outside the Ethiopia and Eritrea domain, associated with different projects working on ancient places. Note that we always use Bm ID for places in the Ethiopia and Eritrea domain. However, using Bm ID for places that are not in our domain can be redundant. In this case, we can use either the wikdata id or the pleiades id depending on the information we have, or both . So we should consider these things when we substitute pleiades id with BM; yes, it affects the information in the texts we have encoded so far.

**Comment by eu-genia** [comment]
In the example mentioned by Dorothea above, Aksum is referred to by its pleiades ID, while it surely is within the EE domain. And there are many examples where Addis Ababa is referred to by its wikidata ID. So it obviously was not the criterion used.
This may be a problem when one needs to collect all attestations of a place.

**Comment by thea-m** [comment]
To come back to this, I would be extremely uncomfortable to edit the pleiades references used in work records, because this was clearly thought through (and, many of them being Greek, is absolutely outside of my personal scope). 
I suggest to
1. consistently use/create BM IDs for places, link with the wikidata ID in the record and consistently add them in wikidata  (as agreed above)
2. edit old records whenever it makes sense 
3. keep the pleiades references in the work records as they are

I don't know if several sameAs alignments are possible, but we could always use the relation skos:exactMatch to add further alignments.

I think that would be a good compromise that would solve the main issue without interfering too much in existing encoding.
